spec/formatting.md

```diff
@@ -57,6 +57,23 @@ nor be made available to _function handlers_.
 > _declarations_ affecting _variables_ referenced by that _expression_
 > have already been evaluated in the order in which the relevant _declarations_
 > appear in the _message_.
+>
+> Implementations and users creating custom functions SHOULD avoid
+> creating _function handlers_ that mutate external program state.
```

Comment:
SHOULD avoid => SHOULD NOT

```suggestion
> Implementations and users SHOULD NOT create _function handlers_
> that mutate external program state,
> particularly since such a _function handler_ can present a remote execution hazard.
```